package.json

```diff
@@ -27,6 +27,7 @@
     "lodash-compat": "^3.9.3",
     "object.assign": "^2.0.1",
     "safe-json-parse": "^4.0.0",
+    "videojs-font": "1.1.0",
```

Comment:
Also, should we be more careful with the compiled dependencies? Lodash I feel safe enough with, but for the others the carat feels pretty trusting of the other project authors and npm to not make any mistakes.